### PR TITLE
pullrequest

### PR DESCRIPTION
**Pull request by ILIK3T0C0D3**
These are just small edits to the Thor2023 code including deleting obsolete files and fixing some problems. Ello and Er are these changes they are the same commit except I accidentally did it twice.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java
Comment (SamueljWilson) [inline src/main/java/frc/robot/Constants.java:1]: Should be able to remove the entire ArmConstants class in the code

Comment (ILIK3T0C0D3) [reply src/main/java/frc/robot/Constants.java:1]: Ok

File: src/main/java/frc/robot/RobotContainer.java
Comment (SamueljWilson) [inline src/main/java/frc/robot/RobotContainer.java:1]: Can also remove the commented code for m_arm and m_wrist

File: src/main/java/frc/robot/commands/Auto.java
Comment (SamueljWilson) [inline src/main/java/frc/robot/commands/Auto.java:1]: Remove Auto.java and DriveTrajectory.java

**Pull request merged**